Resources/Prototypes/Access/misc.yml

```
@@ -52,3 +52,8 @@
   - BlueshieldOfficer # Goobstation - Add BSO Access
   - NanotrasenRepresentative # Goobstation - Add NTR Access
   - Magistrate # Goobstation - Add Magistrate Access
+
+- type: accessGroup
+  id: General
+  tags:
+  - Maintenance
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)